recipes/wdiff/build.sh

```diff
@@ -0,0 +1,7 @@
+set -x
+
+./configure --prefix=${PREFIX}
```

Comment:
Would quote this to protect against whitespace

```suggestion
./configure --prefix="${PREFIX}"
```